### ISSUE TITLE
upgraded to new 0.8.5 and my last sync was block 2,023, 322 

### ISSUE THREAD

**Issue by etherbinge** [ethereum/mist#1280]
#### System information
``````
Version: `0.0.0`
OS & Version: windows/linux/osx
Node type: `eth/geth(default)`
``````

#### Please check the **already existing** issues to keep duplicates at a minimum.
Furthermore several **work-arounds** have been collected in the [Mist-Troubleshooting-Guide](https://github.com/ethereum/wiki/wiki/Mist-Troubleshooting-Guide).

If possible add the following to your report:

- Screenshots

- Check the console, of Mist (`CTRL/CMD + ALT + i`) and take a screenshot

- Log files 
  - go to `menu -> accounts -> backup -> application data`
  - zip and upload `node.log` and all other `node.log.X` files

**Comment by etherbinge** [comment]
I have downloaded purity and upgraded to the new 0.8.5 still telling me that it has been 2 months since my last block.

**Comment by Mordong** [comment]
Just now successfully sync with Parity 1.3.7
https://ethcore.io/parity.html

**Comment by luclu** [comment]
@etherbinge, we just published [release `0.8.6`](https://github.com/ethereum/mist/releases/tag/v0.8.6) which will download **geth** `1.4.18` on-the-fly. 
*Please* update to easily roll with the upcoming [Hard Fork](https://blog.ethereum.org/2016/10/13/announcement-imminent-hard-fork-eip150-gas-cost-changes/) on Tuesday.
